src/data.js

```diff
@@ -17971,6 +17971,26 @@ module.exports = [
       'Problem Solver',
     ],
   },
+  {
+    name: 'Hector Aguilar',
+    description:
+      'A Software Engineer that loves Web development and CLI sofware',
+    url: 'https://bitbyte.blog/uses',
+    emoji: '🐤',
+    computer: 'macos',
```

Comment:
"computer" must be one of [apple, windows, linux, bsd]

Reply:
Thank you for the correction!